sktime/registry/_tags.py

```diff
@@ -1041,6 +1041,40 @@ class capability__predict_proba(_BaseTag):
     }
 
 
+class capability__class_weight(_BaseTag):
+    """Capability: the classifier can use class weights to handle imbalanced data.
+
+    - String name: ``"capability:class_weight"``
+    - Public capability tag
+    - Values: boolean, ``True`` / ``False``
+    - Example: ``True``
+    - Default: ``False``
+
+    This tag applies to classifiers only.
+
+    If the tag is ``True``, the classifier supports class weighting functionality,
+    typically through a ``class_weight`` parameter that allows users to assign
```

Comment:
remove "typically" - the test enforces that specific name (and that is a good idea)